etcd-cluster.yaml

```diff
@@ -22,7 +22,7 @@ SenzaComponents:
           partition: none
           filesystem: tmpfs
           erase_on_boot: false
-          options: size=512m
+          options: size=5120m
```

Comment:
This option creates ram disk. There is no 5GB of memory available on the t2.micro